CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add styling guide documentation for customizing SDK and UI component CSS.
 - Extend and enabled style customizing capabilities on the SDK components.
+- Add `MeetingSessionConfiguration` as a required param to `MeetingManager`'s `join` method. With this change the builders have direct access to `MeetingSessionConfiguration`, this will allow more flexibility to customize the `MeetingSession`.
+- Add `logLevel`, `postLoggerConfig`, `logger`, `enableWebAudio`, and `activeSpeakerPolicy` as optional params to the `MeetingManager`'s `join` method.
```

Comment:
>Add `logLevel`, `postLoggerConfig`, `logger`, `enableWebAudio`, and `activeSpeakerPolicy` as optional params.

This is not accurate. These are not parameters but properties of the optional parameter `options: MeetingManagerJoinOptions`.

Need to mention:
* the optional parameter is `options` which is of type `MeetingManagerJoinOptions`.
* `logLevel`, `postLoggerConfig`, `logger`, `enableWebAudio`, and `activeSpeakerPolicy` are the properties of `MeetingManagerJoinOptions`.